build_logic.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# build_logic.sh - Performs steps to compile and deploy evals
+# exports: evals war file
+# usage: Build evals for local development. Jenkins should be used to deploy to dev and prod
+
+VM=$VM_USR@$VM_ADDRESS
+LIFERAY_SDK=$LIFERAY_DIR/liferay-plugins-sdk-$LIFERAY_SDK_VERSION
+EVALS_WAR=evals-$LIFERAY_SDK_VERSION.1.war
+COMPILED_WAR=$LIFERAY_DIR/bundles/deploy/$EVALS_WAR
+
+# copy evals files to liferay sdk
+rm -rf $LIFERAY_SDK/portlets/evals
+cp -rf $EVALS $LIFERAY_SDK/portlets
```

Comment:
Just curious, why would we need `-f` for this command?

Reply:
We don't